marshmallow/fields.py

```diff
@@ -773,6 +773,8 @@ def _to_string(self, value):
 class Boolean(Field):
     """A boolean field.
 
+    :param set truthy: see below
```

Comment:
That's a bad way to write argument descriptions: when you generate documentation, there will be no "below" available.

Please move below descriptions here and replace those with something like "Default values for constructor arguments".